onnx/defs/logical/defs.cc

```diff
@@ -66,8 +66,8 @@ OPERATOR_SCHEMA(Less)
 
 OPERATOR_SCHEMA(Equal)
     .FillUsing(BinaryLogicDocGenerator("equal"))
-    .TypeConstraint("T", { "tensor(float16)", "tensor(float)", "tensor(double)" },
-                    "Constrains input to float tensors.")
+    .TypeConstraint("T", { "tensor(bool)", "tensor(int32)", "tensor(int64)" },
+                    "Constrains input to integral tensors.")
```

Comment:
I'm confused, doesn't this diff remove support for equality test on floating point?

Reply:
Yeah I think it probably doesn't make too much sense to provide equality test on floats